### PR TITLE
Add execution payload header to light client header capella onwards

### PR DESCRIPTION
**Pull request by g11tech**
Add execution payload header to light client header capella onwards

Part of 
 - https://github.com/ChainSafe/lodestar/issues/5102

### CODE REVIEW

File: packages/beacon-node/src/util/multifork.ts [hunk 1]
```diff
@@ -53,3 +53,25 @@ export function getStateTypeFromBytes(
...
+ * 8 + 32 = 40
+ * ```
+ * class BeaconState(Container):
+ *   genesis_time: uint64 [fixed - 8 bytes]
+ *   genesis_validators_root: Root [fixed - 32 bytes]
+ *   slot: Slot [fixed - 8 bytes]
+ *   ...
+ * ```
+ */
+const SLOT_BYTES_POSITION_IN_LIGHTCLIENTUPDATE = 40;
```
Comment (g11tech) [inline packages/beacon-node/src/util/multifork.ts]: TODO: calc and add corrected position

File: packages/params/src/index.ts [hunk 2]
```diff
@@ -186,6 +194,17 @@ export const FINALIZED_ROOT_GINDEX = 105;
  */
 export const FINALIZED_ROOT_DEPTH = 6;
 export const FINALIZED_ROOT_INDEX = 41;
+
+// TODO: add correct values here
+export const EXECUTION_PAYLOAD_GINDEX = 105;
```
Comment (g11tech) [inline packages/params/src/index.ts]: TODO: cal and update corrected

File: packages/beacon-node/src/util/multifork.ts (hunk 1 above)
Comment (wemeetagain) [reply packages/beacon-node/src/util/multifork.ts]: I think its at offset 0?

should look like:
```
update = {
  beacon: {
    slot
    ...
  }
  ...
}
```
first field in `update` is `beacon`, first field in `beacon` is `slot`

File: packages/params/src/index.ts (hunk 2 above)
Comment (wemeetagain) [reply packages/params/src/index.ts]: looks to be 33

```ts
types.ssz.capella.LightClientUpdate.getPathInfo(['attestedHeader', 'execution']).gindex
// 33
```

Comment (g11tech) [reply packages/params/src/index.ts]: the proof is from the block body to payload, so updated  accordingly

File: packages/beacon-node/src/db/repositories/lightclientBestUpdate.ts [hunk 3]
```diff
@@ -1,14 +1,39 @@
...
 /**
  * Best PartialLightClientUpdate in each SyncPeriod
  *
  * Used to prepare light client updates
  */
-export class BestLightClientUpdateRepository extends Repository<SyncPeriod, altair.LightClientUpdate> {
+export class BestLightClientUpdateRepository extends Repository<SyncPeriod, allForks.LightClientUpdate> {
   constructor(config: IChainForkConfig, db: IDatabaseController<Uint8Array, Uint8Array>) {
+    // Pick some type but won't be used
     super(config, db, Bucket.lightClient_bestLightClientUpdate, ssz.altair.LightClientUpdate);
```
Comment (wemeetagain) [inline packages/beacon-node/src/db/repositories/lightclientBestUpdate.ts:15]: can we use new buckets here and below to make migration easier

Comment (g11tech) [reply packages/beacon-node/src/db/repositories/lightclientBestUpdate.ts:15]: bump the bucket number for this key?

**Review by wemeetagain — APPROVED** [review]

**Review by twoeths — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.6.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.6.0) :tada:

**Pull request merged**